package.json

```diff
@@ -66,12 +65,16 @@
     "lint-staged": "13.0.3",
     "npm-run-all": "4.1.5",
     "prettier": "2.7.1",
+    "sanity": "^3.2.0",
```

Comment:
Let's leave the `peerDependencies` as `"^3.2.0"` but the `devDependencies` as `"3.2.0"`

Reply:
no problem